cmd/kubectl-k8ssandra/register/register_test.go

```diff
@@ -0,0 +1,178 @@
+package register
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+	"runtime"
+	"testing"
+	"time"
+
+	configapi "github.com/k8ssandra/k8ssandra-operator/apis/config/v1beta1"
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/tools/clientcmd"
+	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func TestRegister(t *testing.T) {
+	require.New(t)
+	client1 := (*multiEnv)[0].GetClient("source-namespace")
+	client2 := (*multiEnv)[1].GetClient("dest-namespace")
+
+	if err := client1.Create((*multiEnv)[0].Context, &corev1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: "source-namespace"}}); err != nil {
+		t.Fatal(err)
+	}
+
+	if err := client2.Create((*multiEnv)[1].Context, &corev1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: "dest-namespace"}}); err != nil {
+		t.Fatal(err)
+	}
+
+	buildDir := os.Getenv("BUILD_DIR")
+	if buildDir == "" {
+		_, b, _, _ := runtime.Caller(0)
```

Comment:
suggestion: Use envtest.RootDir() instead of copying the same code here.

Reply:
From recollection this did not work because the level of this directory is different to that of the other call site.

Reply:
RootDir() should be the same, why would this program be called from different path? If I run from a makefile the go test -v, I'm at the call location of Makefile.